themes/edx.org/lms/templates/header.html

```diff
@@ -35,8 +43,12 @@
 % endif
 </%block>
 
+% if uses_pattern_library:
+<header class="header-global ${"slim" if course and not disable_courseware_header else ""}" aria-label="Main" role="banner">
+% else:
 <header class="${"global slim" if course and not disable_courseware_header else "global-new"}" aria-label="Main" role="banner">
```

Comment:
@andy-armstrong after reviewing with @schenedx  we noticed that this needs to change from ```global-new``` to ```header-global-new``` to pick up the updates Sass. Then we should be all set.